launcher/et

```diff
@@ -32,7 +32,7 @@ do
             shift
             ;;
         -h|--help)
-            printf "et (options) [user@]hostname[:port]
+            printf "et (options) [user@]hostname
```

Comment:
Another way to fix this would be to implement quoted IPv6 literals, i.e. `mistertea@[::1]`, but correctly parsing IPv6 literals is painful and best done with library functions rather than regexes.